components/SearchProvince.js

```diff
@@ -80,9 +88,24 @@ function SearchProvince({ onChooseProvince, onSearchGeo, disabled, value }) {
     }
   }, [value])
 
+  const handleBackButton = (e) => {
+    e.preventDefault()
+    router.push('/')
+  }
+
   return (
     <Box onBlur={() => setTimeout(() => setInputFocus(false), 100)}>
       <HStack spacing="2">
+        {backButton && (
+          <Button
+            onClick={handleBackButton}
+            colorScheme="gray"
+            color="gray.600"
```

Comment:
![image](https://user-images.githubusercontent.com/22767465/126150984-d8d49373-6c19-46d4-b923-f7a45aa00fc2.png)

Please remove this color prop, because the colorScheme prop has handled all the colors. So the issue above (button color contrast is bad) doesn't occur.

Reply:
Oops, sorry, sure. Anything else?

Reply:
Nope. That's all.

Reply:
Maybe the problem is the `Icon`. I deleted the color prop inside the `Button` but the back arrow icon color didn't change on dark mode.

![image](https://user-images.githubusercontent.com/3627108/126157399-49419aca-fd8e-48d0-b10a-2d33933e8e75.png)

I'm assigning the existing `dynamicInputColor` as the `ArrowBackIcon` color prop.

![image](https://user-images.githubusercontent.com/3627108/126157959-b759bbe8-a3e0-4e91-a5f9-0c8d7434b4a1.png)

![image](https://user-images.githubusercontent.com/3627108/126158013-745a3337-7a55-4591-82fc-6f939c43364f.png)

what do you think?

Reply:
Hmmmm. Let's do another approach. Just use `IconButton` component instead. See [here](https://chakra-ui.com/docs/form/icon-button).

Reply:
Yes, that will work

Reply:
kind of weird. It doesn't work on map page. But works on home page. (I'll hide it later). I also notice the normal styling like slightly rounded corner is missing.
![image](https://user-images.githubusercontent.com/3627108/126161115-f42fa04c-39b2-442d-9887-0e0538caed52.png)
![image](https://user-images.githubusercontent.com/3627108/126161172-e6d9921f-c00e-4cbc-8c56-411019922dfa.png)
![image](https://user-images.githubusercontent.com/3627108/126161363-7b9e099c-8626-486e-80cf-86a127f43987.png)

Reply:
Hmmm. For the color one, don't forget to add the colorScheme prop. I don't know why the border radius is not rounded, can you investigate it?

Reply:
Hmmm. For the color one, don't forget to add the colorScheme prop. I don't know why the border radius is not rounded, can you investigate it?

Reply:
I added the colorScheme prop but still doesn't work. Looks like because it replaced by color from map.js. I'm checking the other button color by adding a word and got the same issue.
![image](https://user-images.githubusercontent.com/3627108/126168678-d6ee0577-b49a-477e-91a9-c1f1e2428144.png)
![image](https://user-images.githubusercontent.com/3627108/126169120-498dbd2c-083e-4008-abe9-4b5df5013572.png)

Remove the black color of the box on map page. And it's fine.
![image](https://user-images.githubusercontent.com/3627108/126170736-9a8cd692-26f1-4031-bedd-6a019f8fb077.png)

for the border radius. I think add `borderRadius` prop will solve it.
![image](https://user-images.githubusercontent.com/3627108/126170930-4a7e7a0a-9bf7-4a00-a4c4-6f9c10d6b7ad.png)

Reply:
Okay, looks good to me. Please push the new commit.

Reply:
sure ca1cb740f4160b7a43a7785e07fa045f89f16700, 
thank you for your time to review this PR 🙏